src/redux/missions/missionSlice.jsx

```diff
@@ -0,0 +1,74 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+// import axios from 'axios';
```

Comment:
Great job team 🤝

- [ ] As the previous reviewer suggested, kindly remove all commented code from your project as this is regarded as a bad practice.